scylla.go

```diff
@@ -544,6 +544,17 @@ func (p *scyllaConnPicker) Remove(conn *Conn) {
 	}
 }
 
+func (p *scyllaConnPicker) InFlight() int {
+	result := 0
+	for i := range p.conns {
+		idx := int(atomic.AddUint64(&p.pos, 1))
```

Comment:
`idx := int(atomic.AddUint64(&p.pos, 1))` should be calculated once before the loop

Reply:
I changed that to just iterating through p.conns for simplicity